src/navs/index.js

```diff
@@ -2588,6 +2588,12 @@ export const docsMenu = {
                     icon: 'IconCode',
                     color: 'salmon',
                 },
+                {
+                    name: 'Holdouts',
+                    url: '/docs/experiments/holdouts',
+                    icon: 'IconPeople',
+                    color: 'orange',
+                },
```

Comment:
This isnt super important to running an A/B test, so let's put in under "Feature"

Reply:
cdd1993b1